datasetinsights/data/simulation/captures.py

```diff
@@ -122,14 +123,20 @@ def _load_annotations(self, data_root, version):
       """
         annotations = []
         for c_file in glob(data_root, self.FILE_PATTERN):
-            annotation = load_table(
-                c_file,
-                self.TABLE_NAME,
-                version,
-                record_path="annotations",
-                meta="id",
-                meta_prefix="capture.",
-            )
+            try:
+                annotation = load_table(
+                    c_file,
+                    self.TABLE_NAME,
+                    version,
+                    record_path="annotations",
+                    meta="id",
+                    meta_prefix="capture.",
+                )
+            except KeyError:
+                annotation = pd.DataFrame(
+                    {"annotation_definition": [], "capture.id": []}
+                )
```

Comment:
Can you add a unittest for this exception?

Reply:
Sure. do you want a test that `load_table` throws this exception in certain cases or a test that Captures(...) properly handles this case?

Reply:
Would be better to test whether `Captures._load_annotations` can handle this case.